src/plugin_host.jl

```diff
@@ -0,0 +1,174 @@
+start_host() = try wait(nvim_child(HostHandler()).reader) end
```

Comment:
Nice! you can eliminate the `.reader` though, as `wait` is overloaded for the client.